walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -130,6 +132,80 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredWithInvalOptional() {
```

Comment:
What does "Inval" mean here? Is it "Invalid"? In general it's better to not abbreviate any words in method names to avoid ambiguity. Would you mind updating this?